cmake/macos/helpers.cmake

```diff
@@ -65,6 +65,13 @@ function(set_target_properties_obs target)
           PROPERTY XCODE_EMBED_FRAMEWORKS ${SPARKLE})
       endif()
 
+      if(ENABLE_SYPHON)
```

Comment:
Check for the presence of the `mac-syphon` target instead, as the inclusion depends on whether the plugin is configured and not whether some variable is set.

(The variable _might_ trigger the creation of the target, but that should be of no concern for the application target).

Reply:
Changed to check for `TARGET mac-syphon`